src/lib/utils/compliance.ts

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
 import config from '~app/common/config';
 
+const DEV_MODE_ON = 1;
+const DEVELOPER_MODE_FLAG = 'devModeFlag';
```

Comment:
We told with you about making this flag more specific, like `DEV_IGNORE_COUNTRY_RESTRICTION`

Reply:
done